charts/observability-pipeline/templates/NOTES.txt

```diff
@@ -43,3 +50,9 @@ Honeycomb Observability Pipeline is setup and configured.
   {{- end }}
 
 {{- end }}
+
+Your Honeycomb Telemetry Pipeline is ready.
+
+Start sending OTLP data to your pipeline via:
+  HTTP: http://{{ include "honeycomb-observability-pipeline.fullname" . }}-collector:4318
+  gRPC: http://{{ include "honeycomb-observability-pipeline.fullname" . }}-collector:4317
```

Comment:
do both need to be listed? my vote would be to only show http as a starting point